### PR TITLE
EvernoteOAuth set logger

### PR DESCRIPTION
**Pull request by Galunid**
This sets default EvernoteOAuth logger introduced in https://github.com/koreader/evernote-sdk-lua/pull/4 to `DEBUG`, instead of default `print`

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/6308)

### CODE REVIEW

File: plugins/evernote.koplugin/main.lua [hunk 1]
```diff
@@ -405,6 +405,7 @@ function EvernoteExporter:doLogin(username, password)
         domain = self.evernote_domain,
         username = username,
         password = password,
+        logger = DEBUG
```
Comment (Frenzie) [inline plugins/evernote.koplugin/main.lua:408]: Would you mind changing them all to logger.dbg? That `DEBUG` is kind of a blast from the past.

More important, we prefer a trailing comma at the end. ;-)

Comment (Galunid) [reply plugins/evernote.koplugin/main.lua:408]: Sure

File: plugins/evernote.koplugin/main.lua [hunk 2]
```diff
@@ -9,7 +9,7 @@ local UIManager = require("ui/uimanager")
 local Screen = require("device").screen
 local util = require("ffi/util")
 local Device = require("device")
-local DEBUG = require("dbg")
+local logger = require("logger")
```
Comment (Frenzie) [inline plugins/evernote.koplugin/main.lua:12]: Although the order here's a bit random could you put it right above util? ^_^

File: plugins/evernote.koplugin/main.lua [hunk 3]
```diff
@@ -405,6 +405,7 @@ function EvernoteExporter:doLogin(username, password)
         domain = self.evernote_domain,
         username = username,
         password = password,
+        logger = logger.dbg
```
Comment (Frenzie) [inline plugins/evernote.koplugin/main.lua:408]:
```suggestion
        logger = logger.dbg,
```

File: plugins/evernote.koplugin/main.lua [hunk 4]
```diff
@@ -606,11 +607,11 @@ function EvernoteExporter:exportClippings(clippings)
                 NetworkMgr:promptWifiOn()
                 return
             elseif not ok and err then
-                DEBUG("Error occurs when exporting book:", title, err)
+                logger.dbg("Error occurs when exporting book:", title, err)
```
Comment (Frenzie) [inline plugins/evernote.koplugin/main.lua:610]:
```suggestion
                logger.dbg("Error when exporting book:", title, err)
```

**Pull request merged**